SalesforceSDK/Salesforce.Sample.HybridMainPage.Universal/Settings/Config.cs

```diff
@@ -26,39 +26,51 @@
  */
 
 using System;
-using System.Threading.Tasks;
-using Windows.Foundation;
-using Windows.Security.Cryptography;
-using Windows.Storage;
-using Windows.Storage.Streams;
-using Windows.Web;
+using Windows.UI;
+using Salesforce.SDK.Source.Settings;
 
-namespace Salesforce.SDK.Hybrid
+namespace Salesforce.Sample.HybridMainPage.Universal.settings
 {
-    public sealed class StreamUriResolver : IUriToStreamResolver
+    internal class Config : SalesforceConfig
     {
-        public IAsyncOperation<IInputStream> UriToStreamAsync(Uri uri)
+        public override string ClientId
         {
-            string host = uri.Host;
-            int delimiter = host.LastIndexOf('_');
-            string encodedContentId = host.Substring(delimiter + 1);
-            IBuffer buffer = CryptographicBuffer.DecodeFromHexString(encodedContentId);
+            get { return "SfdcMobileWindowsRT"; }
+        }
+
+        public override string CallbackUrl
+        {
+            get { return "sfdc:///axm/detect/oauth/done"; }
```

Comment:
If a default value must be specified, we should use the values from ```bootconfig.json``` for the Mobile SDK sample apps.